images/free_flying.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="14" height="14">
+ <g transform="translate(-187.1 -49.85)">
```

Comment:
please get rid of transformations (eg. use ungroup function in inkscape and then minify SVG again)

I am also missing a white glow as we use for other icons.